store/postgres/src/filter.rs

```diff
@@ -272,6 +272,9 @@ fn store_filter_by_mode<'a>(
         // Is `attribute` equal to some `v` in `query_values`?
         StoreFilter::In(attribute, query_values) => {
             let op = " = ANY (";
+            if query_values.is_empty() {
+                return Ok(add_filter(query, filter_mode, sql("true")));
```

Comment:
I'm curious: what does this do?

Reply:
This seems like it makes `x_in: []` true, which wouldn't be correct.

Reply:
This is similar to writing "1=1", it will always evaluate to true, so it will not filter the results when used in a where clause on its own or when combined with other conditions using `AND`.   
The Postgres query planner should ignore these clauses in the execution plan. 

I was thinking I would use 'false' in the case of filter_mode=Or, but there are some cases where adding `true` or `false` would change the filtering result... so it looks like I need a better solution.